src/internal/model/request/GetEvaluationsRequest.ts

```diff
@@ -1,9 +1,17 @@
 import { SourceID } from '../SourceID'
 import { User } from '../User'
 
+interface UserEvaluationCondition {
+  // This can be a number, but `UserEvaluations.createdAt` is returned as string and is defined as int64, so we use string here.
+  evaluatedAt: string
+  userAttributesUpdated: boolean
+}
+
 export interface GetEvaluationsRequest {
   tag: string
   user: User
   userEvaluationsId: string
   sourceId: SourceID
+  userEvaluationCondition: UserEvaluationCondition
+  sdkVersion: string
```

Comment:
Can you also add the `sdkVersion` in the `RegisterEventsRequest`?

https://github.com/bucketeer-io/javascript-client-sdk/blob/c98ee30f06ece69f223a9bf25d3e6de55d4a8b03/src/internal/model/request/RegisterEventsRequest.ts#L3C18-L4

Reply:
done: [`1bd7cdf` (#91)](https://github.com/bucketeer-io/javascript-client-sdk/pull/91/commits/1bd7cdfa298e08463a486fdd6974b4fc1eb514cb)